core/org.osate.aadl2.modelsupport/src/org/osate/aadl2/modelsupport/resources/OsateResourceUtil.java

```diff
@@ -134,7 +145,7 @@ public static XtextResourceSet createResourceSet() {
 	/**
 	 * method that creates an Xtext-based ResoruceSet (EMF resource set plus synchronization
 	 * It iwll not use the shared/global Osate resource set
-	 * @return
+	 * @return will be removed in 2.5.0
```

Comment:
You didn't deprecate this method.
```suggestion
	 * @deprecated will be removed in 2.5.0
```
The `@Deprecated` annotation should also be added.